packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/efs_guardian/efs_manager.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from pathlib import Path
+from typing import Final
+
+from models_library.api_schemas_efs_guardian import EFS_GUARDIAN_RPC_NAMESPACE
+from models_library.projects import ProjectID
+from models_library.projects_nodes_io import NodeID
+from models_library.rabbitmq_basic_types import RPCMethodName
+from pydantic import NonNegativeInt, parse_obj_as
+
+from ....logging_utils import log_decorator
+from ....rabbitmq import RabbitMQRPCClient
+
+_logger = logging.getLogger(__name__)
+
+
+_DEFAULT_TIMEOUT_S: Final[NonNegativeInt] = 20
+
+
+@log_decorator(_logger, level=logging.DEBUG)
+async def create_project_specific_data_dir(
+    rabbitmq_rpc_client: RabbitMQRPCClient,
+    *,
+    project_id: ProjectID,
+    node_id: NodeID,
+) -> Path:
+    output: Path = await rabbitmq_rpc_client.request(
+        EFS_GUARDIAN_RPC_NAMESPACE,
+        parse_obj_as(RPCMethodName, "create_project_specific_data_dir"),
```

Comment:
minor: the word `specific` is redundant imho

Reply:
The idea here was, that there might be in the future for example `project-shared-data`